src/model/generated/address.model.ts

```diff
@@ -3,6 +3,9 @@ import * as marshal from "./marshal"
 import {RebasingOption} from "./_rebasingOption"
 import {History} from "./history.model"
 
+/**
```

Comment:
nit: Should we add these comments on generated files? 😅

Reply:
The code generation does it for us. Not something I did directly.

Reply:
Ah, makes sense. It appeared on other generated files unrelated to this PR, so I assumed it was a manual addition